workflow/controller/operator.go

```diff
@@ -1116,38 +1116,29 @@ func (woc *wfOperationCtx) assessNodeStatus(pod *apiv1.Pod, node *wfv1.NodeStatu
 		}
 		newDaemonStatus = pointer.BoolPtr(false)
 	case apiv1.PodRunning:
-		if pod.DeletionTimestamp != nil {
```

Comment:
this condition is incorrect - a pod with a deletion timestamp is one we may have caused to be deleted outselver